luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/LuthierValidationUtils.java

```diff
@@ -32,6 +32,25 @@ public static void validateField(
         }
     }
 
+
+    /**
+     * Validates that the given field actually exists. If it doesn't, throws a useful error message.
+     *
+     * @param fieldValue The value to check for existence
+     * @param configEntityType  The type of the config entity whose field we're validating
+     * @param configEntityName The name of the config entity we're currently building
+     * @param fieldName  The name of the field whose value we're validating
+     */
+    public static void validateField(
```

Comment:
This should start to phase out any usage of the function (polymorphism) a few lines above.
I can start a refactor of existing code as we really should be removing all the magic words 
(like `ENTITY_TYPE = "strict physical table"`) everywhere